plugins/neotest/options.nix

```diff
@@ -38,65 +38,71 @@ with lib;
     Minimum log levels.
   '';
 
-  consumers = helpers.defaultNullOpts.mkAttrsOf helpers.nixvimTypes.rawLua "{}" ''
+  # FIXME this is attrsOf (maybeRaw rawLua)!
```

Comment:
`maybeRaw rawLua` ? Does this make sense or am I simply missing something ?

Reply:
I don't think you're missing anything. It sounds like you agree with my FIXME comment.

`mkAttrsOf` wraps the "of" type in `maybeRaw`, so `mkAttrsOf rawLua` is type `nullOr (attrsOf (maybeRaw rawLua))`.

(Separate PR): Do you think attrsOf should try and be "smart" and not maybe-raw-ify already-raw types? Or maybe `mkAttrs'` could have a toggle flag?

Or maybe `*Of {str,strFn,raw}Lua` should be handled separately without using `mkAttrsOf`?

I think we should consider this if/when the option helpers get their "be more explicit" refactor?

Reply:
I have no idea on this...
Let's go ahead and move on with this PR. Maybe you can add this thought to the notepad for us not to forget.

Reply:
I guess for this case we should do `attrsOf strLua`. This would still support any legacy `__raw` types but also allow users to supply string lua.

Reply:
Made it attrsOf strLua, now that #1709 is merged.